opentdf/opentdf
# PLAT-1627 opentdf.io website

there is more UI work not included in this PR. Also not included in this pr is refactoring the site into a `docs` folder or some other non-root directory.

Safari doesn't like `transparent`, so the **Learn More** doesn't look great in Safari, but I couldn't find a solution. I tried `rgb(0,0,0,0)` and rgb(255,255,255,0)` and neither worked, so I'm a little stumped. 

### How to test

1. check out the branch and run `npm install`, then `npm start` in the root directory
2. go to localhost:3000 
3. NOTE: you can also see the site live at my forked github pages, https://cassandrabailey293.github.io/opentdf/ . 

**Get Started** should route to the [quickstart](https://github.com/opentdf/opentdf/tree/main/quickstart#opentdf-quickstart). 
**Learn More** should route to the OpenTDF Github Organization [page](https://github.com/opentdf). 

Github logo in the nav bar should also route there.

Twitter logo should route to https://twitter.com/openTDF . 

API should route to static stoplight api [docs](https://opentdf.stoplight.io/docs/opentdf-full-api-documentation/branches/main/pqb9cavidvql2-open-tdf) . 

<img width="1416" alt="Screen Shot 2022-06-02 at 3 59 17 PM" src="https://user-images.githubusercontent.com/25390369/171745447-5232ba82-d380-42f8-bb2c-57928d1bf43b.png">
